Mail/Views/Thread/MessageView.swift

```diff
@@ -88,12 +87,16 @@ struct MessageView: View {
                     }
 
                     MessageBodyView(
+                        isMessagePreprocessed: $isMessagePreprocessed,
                         presentableBody: $presentableBody,
                         blockRemoteContent: isRemoteContentBlocked,
                         displayContentBlockedActionView: $displayContentBlockedActionView,
                         messageUid: message.uid
                     )
                     .padding(.top, 16)
+                    .onChange(of: isMessagePreprocessed) { _ in
+                        // Keep it for event propagation to work
```

Comment:
Without this, observing does not want to propagate

Reply:
Maybe `.id("\(message.uid) \(isMessagePreprocessed)")`